content/articles/how-to-create-horizontal-scrolling-menu-using-jquery-and-php/index.md

```diff
@@ -0,0 +1,216 @@
+- [Bring menu objects from db through hypertext preprocessor and mysql](#bring-menu-objects-from-db-through-hypertext-preprocessor-and-mysql)
+- [Creating database](#creating-database)
+- [File-structure](#file-structure)
+- [Final menu result](#final-menu-result)
+- [Conclusion](#conclusion)
+
+### Prerequisites 
+This project requires less or no knowledge in creating menu bars, but a little knowledge will be an added advantage but don't you worry, as I will take you through every step. First, ensure you have a text editor installed on your computer. 
+
+You can choose any text editor you wish with notepad, visual studio code, Atom, Sublime Text, etc.
+Also, you need a web server installed. In our case, we will use XAMP(Apache HTTP Server), but you can choose to download your own choice, such as WAMP server, Nginx Web Server, Lighttpd Web Server, etc.
+
+### Significance of utilizing flat menu
+ * They limit time spent looking over what clients should perform to arrive at their ideal substance/content.
+ * Using horizontal scrolling menus saves vertical page space by setting a unique component.
+ * It keeps the menu thing list from wrapping to the accompanying line, which will be acceptable while thinking about vertical dividing and style.
+ * The best strategy for showing a rundown menu of things in a solitary way without using a scrollbar.
+ * It will be reasonable to make a responsive menu interface viable with web and versatile ports.
+
+### Available modules and libraries to make navigation menus
+There are a lot of available libraries and modules out on the web; nonetheless, in this article, we will utilize Jquery with the blend of PHP and HTML to make our menu bar that intermediate and beginner web developers will understand.
+
+### Advantages of exploitation custom code to execute flat looking over
+1. Own customization will consistently work on the code and limit the quantity of documents size.
+1. Contrasted with the devoted third-party modules, the custom code stream will be straightforward.
+1. The menus are adjusted in focus with the utilization of CSS text-align highlight
+
+### Concerning Menu model
+Our example code will be straightforward and fundamental to make a unique parchment menu to create our unique model. JQuery techniques are utilized to deal with flat scrolling and movement.
+On the snap occasion of these controls, the jQuery vivify() technique is called to move the menu things to and fro.
+
+### Code execution to call attention to Menu with flat scrolling
+Navigation symbols :
+ Rightward squiggle & leftward squiggle arrows will be employed in our tutorial.
+We will employ the above symbols to maneuver over the menu, but you can always use any other direction symbols of your choice. With a click, each opposite arrow will help in scrolling over the menu to and fro, thus assisting visitors in fetching menu items quickly on the page.

```

Comment:
This is a section I expect to see some code being executed? Where are the codes?

Reply:
The section explains how the code will be executed through the click of the right and left arrows.
I have elaborated  it to say  ### Execution of the code to show the flat scrolling menu